File: HFSGuardaDiretorio_SharpDevelop_Ruby/catalogador/RotuloRaiz.rb
# 
# * Created by SharpDevelop.
# * User: henrique.souza
# * Date: 07/07/2015
# * Time: 11:01
# *
# * To change this template use Tools | Options | Coding | Edit Standard Headers.
# 
require "mscorlib"
require "System, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a"

module HFSGuardaDiretorio.catalogador
	# <summary>
	# Description of RotuloRaiz.
	# </summary>
	class RotuloRaiz
		def initialize()
		end

		def Rotulo
			return @rotulo
		end

		def Rotulo=(value)
			@rotulo = value
		end
	end
end

File: HFSGuardaDiretorio_GtkPython/catalogador/RotuloRaiz.py
# 
# * Created by SharpDevelop.
# * User: henrique.souza
# * Date: 07/07/2015
# * Time: 11:01
# *
# * To change this template use Tools | Options | Coding | Edit Standard Headers.
# 
from System import *

class RotuloRaiz(object):
	""" <summary>
	 Description of RotuloRaiz.
	 </summary>
	"""
	def __init__(self):

	def get_Rotulo(self):
		return self._rotulo

	def set_Rotulo(self, value):
		self._rotulo = value

	Rotulo = property(fget=get_Rotulo, fset=set_Rotulo)